demos/python_demos/action_recognition/action_recognition_demo/timer.py

```diff
@@ -60,7 +60,7 @@ def time_section(self):
         self.tock()
 
     def __repr__(self):
-        return "{:.2f}ms (±{:.2f}) {:.2f}fps".format(self.avg, self.std, self.fps)
+        return "{:.2f}ms (std: {:.2f}) {:.2f}fps".format(self.avg, self.std, self.fps)
```

Comment:
CI has a weird environment (probably locale) which does not allow to print that symbol. The conclusion I made from that issue is to avoid non ASCII symbols so everyone could process it.

Reply:
OK, however, do you think "std" will be well-understood? IMO, it would be better to use something like "+/-" as the replacement.

Reply:
replaced